test/verify/check-networkmanager-basic

```diff
@@ -41,6 +41,22 @@ class TestNetworkingBasic(NetworkCase):
         b.wait_in_text("#networking-graphs", "Transmitting")
         b.wait_in_text("#networking-graphs", "Receiving")
 
+        ifaces = list(filter(None, m.execute("ip r | grep -o -P '(?<=dev\ ).*(?=\ proto)'").split('\n')))
```

Comment:
What does the `filter()` do?

Listing /sys/class/net may be much easier?
```py
# python3 -c 'import os; print(os.listdir("/sys/class/net"))'
['eth1', 'virbr0-nic', 'lo', 'eth0', 'virbr0']
```


And you may want to filter out `lo` (with either implementation). 

If you don't like /sys/class/net: `ip -brief link show | awk '{print $1}'`

Reply:
> What does the `filter()` do?

There was an empty string in the list, just removes it. 
> 
> Listing /sys/class/net may be much easier?
> 
> ```python
> # python3 -c 'import os; print(os.listdir("/sys/class/net"))'
> ['eth1', 'virbr0-nic', 'lo', 'eth0', 'virbr0']
> ```
> 
> And you may want to filter out `lo` (with either implementation).
> 
> If you don't like /sys/class/net: `ip -brief link show | awk '{print $1}'`

Thanks for making this nice :)